docs/fundamentals/annotations-overview.md

```diff
@@ -121,6 +121,8 @@ api.Resource.Annotations.Add(
 );
 ```
 
+For more information on publishing and deploying annotations, see [Build container images](build-container-images.md).
```

Comment:
I don't know if this is the best segue for publishing and deploying my annotations. Can we link to the new top-level "publishing and deploying" menu instead?